soccer/src/soccer/strategy/agent/position/seeker.cpp

```diff
@@ -187,10 +193,18 @@ double Seeker::eval_point(rj_geometry::Point ball_pos, rj_geometry::Point curren
 
     // Additional heuristics for calculating optimal point
     double ball_proximity_loss = (current_point - ball_pos).mag() * .002;
-    double goal_distance_loss = (8.5 - current_point.y()) * 0.15 + abs(current_point.x()) * 0.15;
+    double goal_distance_loss = (8 - current_point.y()) * 0.15;
```

Comment:
is 8 a field related constant? can we calculate it based on field dims instead? 

are .002 and .15 just parameters for your heuristic? can they be named constants somewhere as well?
options are: function scope, file scope, class scope. doesn't matter to me

Reply:
fixed!